src/argv.ts

```diff
@@ -273,6 +282,10 @@ export class Argv {
         return this.map.get("shellExecutorNoImage") ?? true;
     }
 
+    get defaultImage (): string | null {
+        return this.map.get("defaultImage") ?? null;
```

Comment:
![image](https://github.com/user-attachments/assets/c79702d1-46c7-4847-b43d-332d0d4b593f)
lets move them here instead

Reply:
```
// https://docs.gitlab.com/ee/ci/runners/hosted_runners/linux.html#container-images
return this.map.get("defaultImage") ?? "docker.io/ruby:3.1";
```
yup, this is what i meant which is mostly how we've set the default args for all the arguments.